perf/benchmarkapps/GrpcAspNetCoreServer/Generated/BenchmarkServiceGrpc.cs

```diff
@@ -0,0 +1,200 @@
+// <auto-generated>
+//     Generated by the protocol buffer compiler.  DO NOT EDIT!
+//     source: benchmark_service.proto
+// </auto-generated>
+// Original file comments:
+// Copyright 2015 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// An integration test service that covers all the method signature permutations
+// of unary/streaming requests/responses.
+#pragma warning disable 0414, 1591
+#region Designer generated code
+
+using grpc = global::Grpc.Core;
+
+namespace Grpc.Testing
+{
+    public static partial class BenchmarkService
+    {
+        static readonly string __ServiceName = "grpc.testing.BenchmarkService";
+
+        static void __Helper_SerializeMessage(global::Google.Protobuf.IMessage message, grpc::SerializationContext context)
+        {
+#if !GOOGLE_PROTOBUF_DISABLE_BUFFER_SERIALIZATION
+            var bufferMessage = message as global::Google.Protobuf.IBufferMessage;
+            if (bufferMessage != null)
+            {
+                context.SetPayloadLength(bufferMessage.CalculateSize());
```

Comment:
Missing from Grpc.Tools PR. Needs to be added - https://github.com/grpc/grpc/pull/21653#discussion_r383045853

Reply:
Noted